### ISSUE TITLE
Show dashboard owner on detail page

### ISSUE THREAD

**Issue by jezdez** [getredash/redash#2832]
### Issue Summary

When viewing a dashboard there is no indication of the owner of the dashboard, which may be useful for feedback in an organization for example. We're already showing the owner in the list view, so this could be considered a UI inconsistency.

This was originally filed downstream in mozilla#353.

### Steps to Reproduce

1. Go to a detail page of a dashboard
2. See the name of the user missing

### Technical details:

* Redash Version: 5.0beta
* Browser/OS: Firefox 62, macOS 10.13
* How did you install Redash: Docker

**Comment by arikfr** [comment]
Definitely makes sense. @kocsmy can you suggest a design?

**Comment by kocsmy** [comment]
I have 3 ideas where could we show the dashboard owner:
1. Dashboard header: after the name / tags, we could show a profile picture with the owner. This could be only shown on hover (similar to add tag) so it won't pollute the interface that much.
2. Extra last row after visualizations: we could also use this space to add more information: Created by [profile pics]Arik Fraimovich on 4th of May, 2018, Last edited etc
3. Dashboard info modal which can be reached from the ... menu on the top right corner. We could also place more things here.

I think the option 2 and 3 makes the most sense for me as we could use that space for much more in the future.

**Comment by arikfr** [comment]
Why not a line below the header before the content?

**Comment by kocsmy** [comment]
I don't think this is such an important detail that justifies pushing the content further down.

**Comment by arikfr** [comment]
Then let's go with option #2.

**Comment by kocsmy** [comment]
Ok, I'll get started with it.

**Comment by kocsmy** [comment]
How about a section like this?

![image](https://user-images.githubusercontent.com/2378022/46878258-18436200-ce43-11e8-99fb-2f500a496b1a.png)

- I think this opens up some space for other details for example last edited time.
- I tried it without a background but I kind alike having this slight grey tone.
